### PR TITLE
update tests to use universe v3

### PR DESCRIPTION
**Pull request by tamarrow**
#730

### CODE REVIEW

File: cli/tests/integrations/test_package.py [hunk 1]
```diff
@@ -18,10 +18,15 @@
...
+    # add universe-server with static packages
+    assert_command(
+        ['dcos', 'marathon', 'app', 'add', 'tests/data/universe-v3-stub.json'])
+
     assert_command(
         ['dcos', 'package', 'repo', 'remove', 'Universe'])
-    repo = "https://github.com/mesosphere/universe/archive/cli-test-4.zip"
-    assert_command(['dcos', 'package', 'repo', 'add', 'test4', repo])
+
+    repo = "http://universe.marathon.mesos:8085/repo"
```
Comment (cruhland) [inline cli/tests/integrations/test_package.py]: Factor this value out into a constant? Would make changing this URL a lot easier next time.

File: cli/tests/integrations/test_service.py [hunk 2]
```diff
@@ -17,19 +17,24 @@
...
-        ['dcos', 'package', 'repo', 'remove', 'Universe'])
-    repo = "https://github.com/mesosphere/universe/archive/cli-test-4.zip"
-    assert_command(['dcos', 'package', 'repo', 'add', 'test4', repo])
+    # add universe-server with static packages
+    assert_command(
+        ['dcos', 'marathon', 'app', 'add', 'tests/data/universe-v3-stub.json'])
+
+    exec_command(['dcos', 'package', 'repo', 'remove', 'Universe'])
+
+    repo = "http://universe.marathon.mesos:8085/repo"
```
Comment (cruhland) [inline cli/tests/integrations/test_service.py]: Maybe use the same constant for this too?

File: cli/tests/integrations/test_package.py (hunk 1 above)
Comment (tamarrow) [inline cli/tests/integrations/test_package.py]: This url should never change, only the the docker container referenced in `tests/data/universe-v3-stub.json`

**Comment by cruhland** [comment]
@tamarrow looks like the Windows test failures are related to this change.

**Pull request merged**